core/commands/storage/upload/upload/do_sign_balance.go

```diff
@@ -75,10 +75,7 @@ func balanceHelper(ctx context.Context, configuration *config.Config, signedByte
 	ctx, _ = helper.NewGoContext(ctx)
 	err = grpc.EscrowClient(configuration.Services.EscrowDomain).WithContext(ctx,
 		func(ctx context.Context, client escrowpb.EscrowServiceClient) error {
-			res, err := client.BalanceOf(ctx, &ledgerpb.SignedCreateAccountRequest{
-				Key:       ledgerSignedPubKey.Key,
-				Signature: ledgerSignedPubKey.Signature,
-			})
+			res, err := client.BalanceOf(ctx, ledger.NewSignedCreateAccountRequest(ledgerSignedPubKey.Key, ledgerSignedPubKey.Signature))
```

Comment:
should add the version as 'btfs-escrow-1.0'  as ledger team request

Reply:
already added in go-btfs-common